src/main/rx/expresslrs.c

```diff
@@ -279,7 +279,7 @@ static void unpackChannelDataHybridWide(uint16_t *rcData, volatile elrsOtaPacket
     } else {
         uint8_t bins;
         uint16_t switchValue;
-        if (currTlmDenom > 1 && currTlmDenom < 8) {
+        if (currTlmDenom < 8) {
```

Comment:
was this maybe a typo and it is supposed to check for `currTlmDenom > 0`?

Reply:
`>1 && < 8` does match OpenLRS implementation. 
https://github.com/ExpressLRS/ExpressLRS/blob/225f943b0b5c589cb1a56248332e526da918cbff/src/lib/OTA/OTA.cpp#L418

https://github.com/ExpressLRS/ExpressLRS/pull/2282 . This will reintroduce the bug.

https://github.com/betaflight/betaflight/issues/14291 may be caused by incompatible ELRS version (or implementation) or unexpected telemetry settings. Someone from ELRS may help (ping: @CapnBry)